core/templates/services/playthrough.service.spec.ts

```diff
@@ -82,12 +83,12 @@ describe('PlaythroughService', () => {
     spyOn(Stopwatch, 'create').and.returnValue(mockStopwatch);
   };
 
-  const spyOnStorePlaythrough = (callback: (p: Playthrough) => void = null) => {
+  const spyOnStorePlaythrough = (callback: (p: Playthrough)=> any = undefined )=> {
```

Comment:
Why do you modify this?

Reply:
It was showing null cant be used as value

Reply:
ok can you edit this  to void =null  as my pc is crashin right now and i cant do it

Reply:
I mean I can't assign it a value as it shows error that "_____ can't be assigned it type ____________". Can you suggest what value may i assign it?